test/prx_auth/token_test.exs

```diff
@@ -68,8 +68,8 @@ defmodule PrxAuth.TokenTest do
     now = :os.system_time(:seconds)
     assert expired?(%{}) == false
     assert expired?(%{"iat" => now - 10}) == false
-    assert expired?(%{"exp" => now - 100}) == true
-    assert expired?(%{"exp" => now + 10}) == false
+    assert expired?(%{"iat" => now - 150, "exp" => now - 100}) == true
+    assert expired?(%{"iat" => now - 10, "exp" => now + 10}) == false
```

Comment:
Wait, so is there no test for the current/bad way ID works in here?  (With `exp: 3600`)

I do sort of recall writing this module before I realized PRX/id.prx.org#145 was a problem ... so maybe the client apps aren't even checking `expired?` at all?

Reply:
Not quite - this  `%{"exp" => ... }` pattern was never being matched as far as I can tell, since all calls were being picked up by the `%{"iat" => ..., "exp" => ...}` pattern definition.

You can actually see what's going on here from the way the tests used to be written - without an `iat` present, the code already existed to check the `exp` as a date and not a duration, but it was never being called (if it were, it would always fail, given our tokens).

I am keeping that definition and just starting to call it now, but the tests did not reflect a way that it was actually being invoked anywhere. I have added back tests similar to these, but I still don't expect it to be used as a public API.